packages/forma-36-react-components-datepicker/.gitignore

```diff
@@ -0,0 +1,9 @@
+*.log
+.DS_Store
+node_modules
+.cache
+.rts2_cache_cjs
+.rts2_cache_esm
+.rts2_cache_umd
+.rts2_cache_system
+dist
```

Comment:
I think we can remove this and just move any of the newly ignored files to the root .gitignore